usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -110,6 +110,9 @@ else
     grub-mkconfig -o $grub_conf
 fi
 
+#Finding UUID of filesystem containing /boot 
+grub_boot_uuid=$(df /boot | awk 'END {print $1}' | xargs blkid -s UUID -o value)
```

Comment:
@schabrolles I know of Linux systems where the `/boot` file system is not always mounted. Then the variable would be empty.
What happens in such cases? You would get an invalid grub entry.
Perhaps, foresee a sanity check here?
Gratien